src/components/Button/Button.module.css

```diff
@@ -81,3 +81,11 @@
 .buttonText {
   padding: 0 var(--spacing-padding-xs, 4px);
 }
+
+.primaryOutlined {
+  background-color: var(--palette-action-secondary-main, #ffffff) !important;
```

Comment:
we can't use backgrounds like that. These are things that then cannot be themed

Reply:
You mean we can't use backgrounds at all? There are also other usages in this CSS. 
How should I do it?

Reply:
can u add a screenshot to show the problem?

Reply:
![image](https://github.com/user-attachments/assets/063c8562-4158-456c-8781-20498ed0f80d)
![image](https://github.com/user-attachments/assets/7df5c053-2c1f-4b70-8d46-fd05041b9958)

As you can see in the first image the background is transparent, but it's supposed to be white.
The same applies to the Danger Outlined button

Reply:
Ok I got your point. The problem is that our hierarchies, types and variants are not aligned 1:1 to antd ones, and our mapping has some issues, for example, we use ghost to obtain outline, which is wrong because it also make the background transparent.

The correct fix is to rewrite a complete mapping from our types to antd types, in a way that we replicate all the desired behaviors